### DISCUSSION TITLE
PDF file offile

### DISCUSSION THREAD

**Discussion by EvgenyPrikhodko** [xamarin/Xamarin.Forms#14188] [Q&A]
Hi. 
How solve this task? In offline mode we must see PDF file. 
WebView vs PDFJS? 
Does anyone know of a nuget for this task?

**Comment by Jakar510** [comment]
You have to use PDFJS for android, iOS is not needed as WebView includes the functionality. 

Copy the PDFJS folder to the android assets folder, then in Visual Studio, show hidden files, and include the entire folder in the project. then pass the file path to the following URI: $"file:///android_asset/pdfjs/web/viewer.html?file={Path}";

You will need a custom renderer to do all this. 

Note: you must download the file locally first, so you can get the file path to pass to the URI.
